diffrax/_solver/milstein.py

```diff
@@ -42,13 +43,15 @@ class StratonovichMilstein(AbstractStratonovichSolver):
         Note that this commutativity condition is not checked.
     """  # noqa: E501
 
-    term_structure: ClassVar = MultiTerm[tuple[ODETerm, AbstractTerm]]
+    term_structure: ClassVar = MultiTerm[
+        tuple[ODETerm, AbstractTerm[AbstractBrownianPath]]
```

Comment:
I think this needs updating to use the return type, not the type of the control path itself.

(Which in this case I think means we can just keep the original unparameterised
```python
term_structure: ClassVar = MultiTerm[tuple[ODETerm, AbstractTerm]]
```
)